Install-DVWA.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
+clear
+
+# Verify if the user is root
+if [ "$EUID" -ne 0 ]; then
+    echo -e "\e[91mThis script must be run by the root user.\e[0m"
+    exit 1
+fi
+
+# Function to check the existence of a program
+check_program() {
+    command -v "$1" >/dev/null 2>&1 || {
+        echo -e >&2 "\033[91m$1 is not installed. Installing it now..."
+        apt install -y "$1" > /dev/null 2>&1
+    }
+}
+
+# ASCII Art
+echo -e "\033[96m\033[1m
+⠄⠄⠄⠄⠄⠄⠄⢀⣠⣶⣾⣿⣶⣦⣤⣀⠄⢀⣀⣤⣤⣤⣤⣄⠄⠄⠄⠄⠄⠄
+⠄⠄⠄⠄⠄⢀⣴⣿⣿⣿⡿⠿⠿⠿⠿⢿⣷⡹⣿⣿⣿⣿⣿⣿⣷⠄⠄⠄⠄⠄
+⠄⠄⠄⠄⠄⣾⣿⣿⣿⣯⣵⣾⣿⣿⡶⠦⠭⢁⠩⢭⣭⣵⣶⣶⡬⣄⣀⡀⠄⠄
+⠄⠄⠄⡀⠘⠻⣿⣿⣿⣿⡿⠟⠩⠶⠚⠻⠟⠳⢶⣮⢫⣥⠶⠒⠒⠒⠒⠆⠐⠒
+⠄⢠⣾⢇⣿⣿⣶⣦⢠⠰⡕⢤⠆⠄⠰⢠⢠⠄⠰⢠⠠⠄⡀⠄⢊⢯⠄⡅⠂⠄
+⢠⣿⣿⣿⣿⣿⣿⣿⣏⠘⢼⠬⠆⠄⢘⠨⢐⠄⢘⠈⣼⡄⠄⠄⡢⡲⠄⠂⠠⠄
+⣿⣿⣿⣿⣿⣿⣿⣿⣿⣷⣥⣀⡁⠄⠘⠘⠘⢀⣠⣾⣿⢿⣦⣁⠙⠃⠄⠃⠐⣀
+⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣋⣵⣾⣿⣿⣿⣿⣦⣀⣶⣾⣿⣿⡉⠉⠉
+⣿⣿⣿⣿⣿⣿⣿⠟⣫⣥⣬⣭⣛⠿⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡆⠄
+⣿⣿⣿⣿⣿⣿⣿⠸⣿⣏⣙⠿⣿⣿⣶⣦⣍⣙⠿⠿⠿⠿⠿⠿⠿⠿⣛⣩⣶⠄
+⣛⣛⣛⠿⠿⣿⣿⣿⣮⣙⠿⢿⣶⣶⣭⣭⣛⣛⣛⣛⠛⠛⠻⣛⣛⣛⣛⣋⠁⢀
+⣿⣿⣿⣿⣿⣶⣬⢙⡻⠿⠿⣷⣤⣝⣛⣛⣛⣛⣛⣛⣛⣛⠛⠛⣛⣛⠛⣡⣴⣿
+⣛⣛⠛⠛⠛⣛⡑⡿⢻⢻⠲⢆⢹⣿⣿⣿⣿⣿⣿⠿⠿⠟⡴⢻⢋⠻⣟⠈⠿⠿
+⣿⡿⡿⣿⢷⢤⠄⡔⡘⣃⢃⢰⡦⡤⡤⢤⢤⢤⠒⠞⠳⢸⠃⡆⢸⠄⠟⠸⠛⢿
+⡟⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠁⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⢸
+\033[0m"
+
+echo -e "\033[96mWelcome to the DVWA installer!\033[0m"
+
+echo
+echo
+# Function to display instructions in English
+show_english_instructions() {
+	echo -e "\e[96mUpdating repositories...\e[0m"
+	apt update > /dev/null 2>&1
+	clear
+    
+	echo -e "\e[96mVerifying and installing necessary dependencies...\e[0m"
+    
+    check_program apache2
+    check_program mariadb-server
+    check_program mariadb-client
+    check_program php
+    check_program php-mysqli
+    check_program php-gd
+    check_program libapache2-mod-php
+    check_program git
+    sleep 2
+	clear
+    
+	echo -e "\e[96mDownloading DVWA from GitHub...\e[0m"
+    git clone https://github.com/ethicalhack3r/DVWA.git /var/www/html/DVWA
+	sleep 2
+	clear
+    
+	echo -e "\e[96mStarting MySQL...\e[0m"
+    systemctl start mysql.service
+	sleep 2
+	clear
+    
+	echo -e "\e[96mConfiguring the database for DVWA...\e[0m"
+    mysql -u root -e "CREATE DATABASE IF NOT EXISTS dvwa;"
+    mysql -u root -e "CREATE USER 'dvwa'@'localhost' IDENTIFIED BY 'abc123';"
+    mysql -u root -e "GRANT ALL PRIVILEGES ON dvwa.* TO 'dvwa'@'localhost';"
+    mysql -u root -e "FLUSH PRIVILEGES;"
+	sleep 2
+	
+    echo -e "\e[96mConfiguring DVWA...\e[0m"
+    cp /var/www/html/DVWA/config/config.inc.php.dist /var/www/html/DVWA/config/config.inc.php
+    sed -i "s/\(\$_DVWA\[ 'db_password' \] = '\).*\('\)/\1abc123\2/" /var/www/html/DVWA/config/config.inc.php
+    sleep 2
+
+    echo -e "\e[96mConfiguring permissions...\e[0m"
+    chown -R www-data:www-data /var/www/html/DVWA
+    chmod -R 755 /var/www/html/DVWA
+    sleep 2
+
+    echo -e "\e[96mConfiguring PHP...\e[0m"
+    php_config_file="/etc/php/$(php -r 'echo PHP_MAJOR_VERSION . "." . PHP_MINOR_VERSION;')/apache2/php.ini"
```

Comment:
I think this is ok for now, but need to watch out in case the default install becomes the fpm version.

If that happens, other things will probably break, just highlighting it in case.